### PR TITLE
Remove old stuff

### PR DESCRIPTION
**Pull request by theory**
* Remove straggler references to versions of Postgres prior to 9.1, including in the README
* Update URLs to use https wherever possible (and fix some broken links)
* Update copyright date

Damn, @nasbyj, seems like we have a bunch of PRs to review if we want to do another release. Which we should probably consider soon.

### CODE REVIEW

**Comment by theory** [comment]
Also wondering how many of the extra expected files we can eliminate now.

File: sql/pgtap.sql.in [hunk 1]
```diff
@@ -1852,7 +1852,7 @@ RETURNS text AS $$
     ), $2);
 $$ LANGUAGE SQL immutable;
 
--- Borrowed from newsysviews: http://pgfoundry.org/projects/newsysviews/
+-- Borrowed from newsysviews: sFoundry/newsysviews/
```
Comment (wolfgangwalther) [inline sql/pgtap.sql.in:1855]: that seems broken

**Comment by theory** [comment]
Care to give this a review, @nasbyj?

**Review by nasbyj — APPROVED** [review]
LGTM

**Pull request merged**